Header header
time hwts
uint8 id
int16 current
int16 rpm

